net/http/authz/authorizer.go

```diff
@@ -41,7 +44,7 @@ func NewAuthorizer(rdb *raft.Service, prefix string, policyMap map[string][]stri
 func (a *Authorizer) Authorize(req *http.Request) error {
 	policies := a.policyByRoute[strings.TrimRight(req.RequestURI, "/")]
 	if policies == nil || len(policies) == 0 {
-		return errors.New("missing policy on this route")
+		return ErrMissingPolicy
```

Comment:
we don't _need_ it (and I left it partly in case someone asked me to return something other than a 500). will remove!